app/__init__.py

```diff
@@ -30,5 +30,10 @@ def create_app(test_config=None):
     migrate.init_app(app, db)
 
     # Register Blueprints here
+    from .routes import tasks_bp
+    from .routes import goals_bp
```

Comment:
you can combine the imports 
```suggestion
    from .routes import tasks_bp, goals_bp
```